packages/manager/modules/pci/src/projects/project/project.less

```diff
@@ -1,6 +1,18 @@
 @background-color: rgb(254, 234, 134);
 @text-color: rgb(139, 97, 17);
 @link-color: rgb(0, 80, 215);
+@alpha-background-color: #fbff47;
+@alpha-font-color: #00185e;
+
+.oui-status_alpha {
```

Comment:
I've recently updated the color from the badge `beta`.

Since we are relying on a override maybe it could be moved right next the `beta` styles.

Related to: https://github.com/ovh/manager/pull/2296/files#diff-a7b979d7d31f21e2905d7f31c8b3823bR186

So changes can be applied here: `packages/manager/apps/public-cloud/src/assets/theme/default/_sidebar.less`.

Is that make sense for you?